lms/static/js/learner_dashboard/models/course_card_model.js

```diff
@@ -224,10 +223,10 @@ class CourseCardModel extends Backbone.Model {
   }
 
   updateCourseRun(courseRunKey) {
-    const selectedCourseRun = _.findWhere(this.get('course_runs'), { key: courseRunKey });
+    const selectedCourseRun = this.get('course_runs').find(run => run.key === courseRunKey);
     if (selectedCourseRun) {
       // Update the current context to set the course run to the enrolled state
-      _.each(this.context.course_runs, (run) => {
+      this.context.course_runs.forEach((run) => {
```

Comment:
ditto to line 74 comment

This one seems a bit more like a style/nit call.

```javascript
this.context.course_runs.forEach( run => 
  Object.assign( run, {
    is_enrolled: run.key === selectedCourseRun.key ? true : run.is_enrolled,
  });
```

Reply:
I think you could even make this more terse by doing

```javascript
is_enrolled: run.key === selectedCourseRun.key || run.is_enrolled
```

I believe what will happen is that if `run.key !== selectedCourseRun.key` then use `run.is_enrolled` else `true`.

Reply:
Done.